py/picca/io.py

```diff
@@ -1307,17 +1307,17 @@ def read_from_minisv_desi(in_dir, catalog, pk1d=None, usesinglenights=False, use
                     data[in_healpixs[w_t]] = []
                 #this might be slow, but would coadd objects with the same targetid even if on multiple tiles
                 do_append=True
+#                if forest.log_lambda is None:
+#                    breakpoint()
                 for index_coadd,forest_existing in enumerate(data[in_healpixs[w_t]]):
                     if forest_existing.thingid==forest.thingid:
-                        forest_existing.coadd(forest)
+                        forest_existing = forest_existing.coadd(forest)
```

Comment:
does this work? or would this be destroying inter-tile coadding and we need to use `data[in_healpixs[w_t]]=forest_existing.coadd(forest)` instead?

Reply:
the method call is ok here, as in this case forest_existing is a pre-existing forest (and actually the `forest_existing=` is unneccessary), but need to check for validity when adding forests to data to ensure this.